### PR TITLE
fixing formatting issues

### PR DESCRIPTION
**Pull request by yomiodeleye**
### What

Fix formatting of National Statistics paragraph on the release page

### How to review

image

### Who can review
FD

### CODE REVIEW

**Review by rafahop — CHANGES_REQUESTED** [review]
Looks like we are removing the line at the end of the files, can we revert that and reduce the diff please?

File: assets/locales/service.en.toml [hunk 1]
```diff
@@ -298,7 +298,11 @@ one = "These statistics have been given National Statistics status by the Office
 
 [AboutDataNationalStatisticsBody2]
 description = "They meet the highest standards of trustworthiness, quality and public value. This means that they:"
-one = " They meet the highest standards of trustworthiness, quality and public value. This means that they:"
+one = " They meet the highest standards of trustworthiness, quality and public value."
```
Comment (rafahop) [inline assets/locales/service.en.toml:301]: Should we amend de description and remove the `This means that they:` bit

Comment (yomiodeleye) [reply assets/locales/service.en.toml:301]: this is done.

**Review by rafahop — COMMENTED** [review]
Please update the description in the Welsh dictionary too

**Review by rafahop — APPROVED** [review]

**Pull request merged**